gdc_client/utils.py

```diff
@@ -0,0 +1,14 @@
+from urllib import urlencode
+
+
+def url_with_params(path, *params, **kwparams):
+    final_params = []
+    if params:
+        final_params.append('&'.join(params))
```

Comment:
Isn't the ampersand taken care of in the return statement? I think this might make && depending on how it is called.

Reply:
There are 2 parts here. Regular params will look like `a&b&c` and key word params like `d=e&f=g`, so the `final_params == ['a&b&c', 'd=e&f=g']`. E.g.

    >>> from gdc_client.utils import url_with_params
    >>> url_with_params('http://example/com/some/path', 'latest', 'tarfile', foo='bar', baz='foo')
    'http://example/com/some/path?latest&tarfile&foo=bar&baz=foo'